.github/workflows/foundry.yml

```diff
@@ -0,0 +1,34 @@
+name: test
```

Comment:
Interesting, kind of curios to see why this workflow is not running on this PR 🤔

Reply:
Oh I see was missing this
```
on:
  push:
    branches: ['main']
  pull_request:
    branches: ['main']
```